extensions/amp-analytics/0.1/vendors/infonline_anonymous.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {jsonLiteral} from '../../../../src/json';
+
+const INFONLINE_ANONYMOUS_CONFIG = jsonLiteral({
+  'vars': {
+    'ap': '1',
+  },
+  'transport': {'beacon': false, 'xhrpost': false, 'image': true},
+  'requests': {
+    'pageview':
+      '${url}?' +
+      'ap=${ap}' +
+      '&dn=${dn}' +
+      '&cn=${cn}' +
+      '&st=${st}' +
+      '&co=${co}' +
+      '&cp=${cp}' +
+      '&au=${canonicalUrl}' +
+      '&ar=${documentReferrer}' +
+      '&ash=${screenHeight}' +
+      '&asw=${screenWidth}' +
+      '&tp=pageview',
+    'event':
+      '${url}?' +
+      'ap=${ap}' +
+      '&dn=${dn}' +
+      '&cn=${cn}' +
+      '&ev=${ev}' +
+      '&st=${st}' +
+      '&co=${co}' +
+      '&cp=${cp}' +
+      '&au=${canonicalUrl}' +
+      '&ar=${documentReferrer}' +
+      '&ash=${screenHeight}' +
+      '&asw=${screenWidth}' +
+      '&tp=event',
+  },
+  'triggers': {
+    'pageview': {
+      'on': 'visible',
+      'request': 'pageview',
+    },
+  },
+});
+
+export {INFONLINE_ANONYMOUS_CONFIG};
```

Comment:
same: This file is no longer needed. Now we rely on the config from the .json  file